openbas-front/src/admin/components/simulations/simulation/Index.tsx

```diff
@@ -173,6 +174,9 @@ const Index = () => {
       setLoading(false);
     });
   });
+  useDataLoader(() => {
+    dispatch(fetchScenario(exercise.exercise_scenario));
```

Comment:
Do we really need to fetch the entire scenario just to display a name? Why not have a button that says 'Go to Parent Scenario' instead?

Reply:
I've checked with product re: UX and design and the outcome was that it was an ok spot with displaying the scenario name. Fetching a single scenario is probably fine in this instance (i.e. it's not a collection).